clients/src/main/resources/common/message/ConsumerGroupHeartbeatResponse.json

```diff
@@ -39,7 +39,7 @@
     { "name": "ErrorMessage", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
       "about": "The top-level error message, or null if there was no error." },
     { "name": "MemberId", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
-      "about": "The member id generated by the coordinator. Only provided when the member joins with MemberEpoch == 0." },
+      "about": "The member id is generated by the consumer and provided by the consumer for all requests starting from version 1. In version 0, however, the member ID is generated by the group coordinator." },
```

Comment:
`In version 0, however, the member ID is generated by the group coordinator.`

This description is inaccurate; in version 0, users can also generate a member ID.